Lib/test/test_ast.py

```diff
@@ -756,6 +756,12 @@ def test_assignment_expression_feature_version(self):
         with self.assertRaises(SyntaxError):
             ast.parse('(x := 0)', feature_version=(3, 7))
 
+    def test_invalid_major_feature_version(self):
```

Comment:
How about just make one test not equal to 3! I know this will change in 4 but so will this fix.